addons/web/static/src/legacy/js/public/public_widget.js

```diff
@@ -247,6 +247,7 @@ export const PublicWidget = Class.extend(mixins.PropertiesMixin, ServicesMixin,
      * @param {Object} [options]
      */
     init: function (parent, options) {
+        console.log(parent);
```

Comment:
u forgot to remove this

Reply:
Yes sorry about that :sweat_smile: 
i was just doing some RD regarding Public widgets :slightly_smiling_face: